spec/std/base64_spec.cr

```diff
@@ -1,6 +1,18 @@
 require "spec"
 require "base64"
 require "crystal/digest/md5"
+require "../support/string"
+
+# rearrange parameters for `assert_prints`
+{% for method in %w(encode strict_encode urlsafe_encode) %}
+  private def base64_{{ method.id }}(io : IO, data, *args)
```

Comment:
Is this overload being used?

Reply:
The `assert_prints` macro rewrites the call argument to use this overload:

```crystal
assert_prints base64_encode(foo)

# expands to the following calls:
base64_encode(foo)
String.build { |io| base64_encode(io, foo) }
string_build_via_utf16 { |io| base64_encode(io, foo) }
```

Reply:
We should certainly swap the order of parameters. Apparently, this was missed by #5916